pilot/cmd/pilot-agent/main.go

```diff
@@ -468,6 +477,11 @@ func init() {
 	proxyCmd.PersistentFlags().BoolVar(&controlPlaneBootstrap, "controlPlaneBootstrap", true,
 		"Process bootstrap provided via templateFile to be used by control plane components.")
 
+	proxyCmd.PersistentFlags().StringVar(&k8sServiceAccountJWTPath, "k8sServiceAccountJWTPath", "",
```

Comment:
I thought you're adding env variables - why the flags ? No need to have both.

Reply:
oh, this is for pilot/mixer bootstrap(as they use a different bootstrap file); will remove this change in this PR and send it with pilot/mixer bootstrap change.